### DISCUSSION TITLE
Topological correct colormap warning when plotting orientations

### DISCUSSION THREAD

**Discussion by hannahcunning** [mtex-toolbox/mtex#1500] [Ask Anything]
I have an EBSD map containing (primarily) Calcite and Dolomite. I am wanting to plot the orientations of both phases but whenever I want to plot the orientations of Dolomite I get the Warning: "Warning: Not a topological correct colormap! Please use the point group "3" "
This happens if I am plotting Dolomite by itself or with other phases. I do not get it when I plot Muscovite, Quartz or Apatite.

Code I use for plotting:
figure,
plot(ebsd('Calcite'),ebsd('Calcite').orientations,'coordinates','on')
hold on
plot(ebsd('Dolomite'),ebsd('Dolomite').orientations,'coordinates','on')
hold off

**Comment by kilir** [comment]
Hi,
dolomite has has space group R-3. There is no way to continuously colorcode directions on a sphere (with the crystalsymmetry -3), it is simply not possible. Hence the warning.

If you look at it, you see there are jumps over the "equator". 

```
cs = crystalSymmetry('-3')
ck = ipfHSVKey(cs)
plot(ck,'3D')
```

Hope this clarifies why there is a warning.

Cheers,
Rüdiger
